sorc/enkf_chgres_recenter_nc.fd/src/makefile

```diff
@@ -1,6 +1,6 @@
 SHELL=  /bin/sh
 
-LIBS= $(FV3GFS_NCIO_LIB) $(BACIO_LIB4) $(W3NCO_LIB4) $(IP_LIB4) $(SP_LIB4) -L$(NETCDF)/lib -lnetcdff -lnetcdf -lhdf5_hl -lhdf5 -lz
+LIBS= $(FV3GFS_NCIO_LIB) $(BACIO_LIB4) $(W3NCO_LIB4) $(IP_LIB4) $(SP_LIB4) -L$(NETCDF)/lib -lnetcdff -lnetcdf -lz # -lhdf5_hl -lhdf5
```

Comment:
Remove the commented libs

Reply:
No problem. I will delete them in the next commit